packages/babel-preset-env/package.json

```diff
@@ -60,8 +60,8 @@
     "@babel/core": "7.0.0-beta.52",
     "@babel/helper-fixtures": "7.0.0-beta.52",
     "@babel/helper-plugin-test-runner": "7.0.0-beta.52",
+    "caniuse-db": "1.0.30000851",
```

Comment:
could we maybe try some auto-updates for `caniuse-db` & `electron-to-chromium`? maybe just a greenkeeper integration (as fully automatic probably would have numerous flaws)?

Reply:
I've had good success with using https://renovatebot.com, which is very customizable and supports monorepos.

Reply:
Yeah we pinned the version of compat-table as well so this is good to me, unless we want to allow people to update before we do via peerDep or override if we are slow to update.

Reply:
Although we'd need to get renovate to update dependencies *and* run the update script. I'm not sure if that'd be possible or not.